src/pages/UserEnrollments/Enrollment.css

```diff
@@ -0,0 +1,77 @@
+.enrollment {
+  width: 100%;
+  box-shadow: 0 1px 4px 0 #C9D3DF;
+  background-color: var(--white);
+}
+
+.enrollment-header {
+  display: flex;
+  justify-content: space-between;
+  flex-flow: wrap;
+  align-items: center;
+  padding: 1em;
+  border-bottom: 1px solid var(--light-grey);
+}
+
+.enrollment-header .fs {
+  display: flex;
+  align-items: center;
+  font-size: 1.25em;
+  font-weight: 600;
+}
+
+.enrollment-header .fs img {
+  height: 40px;
+  margin: 0 0.5em;
+}
+
+.title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-flow: wrap;
+}
+
+.intitule {
+  font-weight: 600;
+  font-size: large;
+}
+
+.enrollment-body {
+  padding: 1em;
+}
+
+.enrollment-footer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 2em;
+}
+
+.tag {
```

Comment:
`tag` is a bit confusing as a name as it is not a tag we are dealing with but enrollment status